letsencrypt-win-simple/Plugin/OverviewPlugin.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Serilog;
+using Microsoft.Web.Administration;
+
+namespace LetsEncrypt.ACME.Simple
+{
+    public class OverviewPlugin : Plugin
+    {
+        private const string ClientName = "letsencrypt-win-simple";
+
+        public override string Name => "OverviewPlugin";
+
+        public override void PrintMenu()
+        {
+            Console.WriteLine(" O: Get overview of all bindings, protocols, renewals and certificates");
+        }
+
+        public override void HandleMenuResponse(string response, List<Target> targets)
+        {
+            if (response == "o")
+            {
+                Console.WriteLine("Running Overview Plugin");
+                Log.Information("Running Overview Plugin");
+
+                GetIisSites();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetRenewals();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetCertificates();
+                Console.WriteLine();
+            }
+        }
+
+        public static void GetIisSites()
```

Comment:
Is the goal to only to list the sites/ hosts that the app will see? If so, then you should instantiate a new plugin object and call the IISPlugin's GetSites.

If not, I don't see the point in listing every IIS site and every binding in the app.

Also, you need to consider that listing these can be long and by default the command prompt only shows 50 lines. So you need to do some pagination like is done in the main program for listing sites. Make sure you use the hostsPerPage setting.